### PR TITLE
Adding filebrowser package

### PR DESCRIPTION
**Pull request by utieyin**
filebrowser:2.31.2 - new package

#### For new package PRs only

- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates

### CODE REVIEW

File: filebrowser.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      ldflags: |
+        -X github.com/filebrowser/filebrowser/v2/version.Version=${{package.version}}
+        -X github.com/filebrowser/filebrowser/v2/version.CommitSHA=$(git rev-parse HEAD)
+      output: ${{package.name}}
+
+  - name: copy files
+    pipeline:
+      - runs: |
+          cp docker_config.json ${{targets.contextdir}}/.filebrowser.json
+          cp healthcheck.sh ${{targets.contextdir}}/
```
Comment (Dentrax) [inline filebrowser.yaml:47]: Just checking - do we need `-oci-entrypoint` or `-compat` subpackage here?

File: filebrowser.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+test:
+  environment:
+  pipeline:
+    - runs: |
+        filebrowser --help
+        filebrowser version | grep ${{package.version}}
```
Comment (Dentrax) [inline filebrowser.yaml:54]: Is it possible to expand the tests a bit?

File: filebrowser.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      - runs: |
+          npm install --frozen-lockfile
+          NODE_ENV=production npm run build
+
+  - uses: go/build
+    with:
+      modroot: .
+      packages: .
+      ldflags: |
+        -X github.com/filebrowser/filebrowser/v2/version.Version=${{package.version}}
```
[the remark this answers is not in the document]
Comment (utieyin) [reply filebrowser.yaml:39]: @Dentrax Am not entirely sure what the ask is here but see command and result
```
docker run --rm filebrowser/filebrowser version
File Browser v2.31.2/129a4fd3
```

File: filebrowser.yaml (hunk 2 above)
Comment (utieyin) [reply filebrowser.yaml:66]: Checking the health status as additional test.

File: filebrowser.yaml (hunk 1 above)
Comment (utieyin) [reply filebrowser.yaml:49]: Nice catch, added it now

**Review by kranurag7 — APPROVED** [review]
thanks for working on it!! looks good.

**Pull request merged**